### PR TITLE
Ch deploy heroku 137689861

### PR DESCRIPTION
**Pull request by gochieng**
#### What does this PR do?
Deploys the application to heroku.
#### Description of Task to be completed?
Deployment of the application to heroku and ensure it is accessible online.
#### How should this be manually tested?
The application can be accessed via https://wger-elcuatro.herokuapp.com/en/software/features
#### What are the relevant pivotal tracker stories?
Story number 137689861

### CODE REVIEW

**Review by austinroy — COMMENTED** [review]
Awesome job on the task guys, I don't see much to improve on.

**Comment by gochieng** [comment]
Thank you Austin.

File: manage.py [hunk 1]
```diff
@@ -11,10 +11,10 @@
 if __name__ == "__main__":
 
     # If user passed the settings flag ignore the default wger settings
-    # if not any('--settings' in s for s in sys.argv):
-    #     setup_django_environment(get_user_config_path('wger', 'settings.py'))
+    if not any('--settings' in s for s in sys.argv):
+        setup_django_environment(get_user_config_path('wger', 'settings.py'))
```
Comment (andela-andrewmakenzi) [inline manage.py:15]: great , definitely going to setup our environment

Comment (ghost) [reply manage.py:15]: thanks you all for the reviews

**Review by tonerdo — APPROVED** [review]

**Pull request merged**